pom.xml

```diff
@@ -85,6 +87,19 @@
             <groupId>uk.gov.companieshouse</groupId>
             <artifactId>structured-logging</artifactId>
             <version>${structured-logging.version}</version>
+            <!--Overriding as the org.apache.commons in structured-logging.version contains vulnerability-->
```

Comment:
Which vulnerability?
Knowing that will help people to remove this override at a future date.